_sections/home-english.md

```diff
@@ -170,6 +170,12 @@ In general, choice names should be unique within a single-choice list. If two ch
 | ===========             |
 | settings                |
 
+#### Cascading selects
+
+It's common to have multiple connected select questions where earlier ones filter the options for later ones. For example, a form might start out by asking the respondent's state, then their district, and finally their city. An alternative would be to directly have the user select their city name from a list of all cities, but the city list would be very long and wouldn't work for city names that are repeated in different districts or states.
```

Comment:
I would recommend having something more like, "It is common to have multiple connected multiple choice questions, where later questions are filtered by earlier question responses." I think select questions is not as clear as multiple choice or select_one and select_multiple, especially to newer form authors.